src/Symfony/Component/Security/Acl/Dbal/AclProvider.php

```diff
@@ -173,7 +173,8 @@ public function findAcls(array $oids, array $sids = array())
             }
 
             // Is it time to load the current batch?
-            if ((self::MAX_BATCH_SIZE === count($currentBatch) || ($i + 1) === $c) && count($currentBatch) > 0) {
+            $intCurrentBatchesCount = count($currentBatch);
```

Comment:
The `int` prefix should be removed because Symfony doesn't follow this practice.